ntbs-service/Services/AuthorizationService.cs

```diff
@@ -170,13 +174,21 @@ private bool UserBelongsToResidencePhecOfNotification(Notification notification)
             return _userPermissionsFilter.UserBelongsToPHEC(phecCode);
         }
 
+        public async Task<IQueryable<Notification>> FilterSharedNotificationsByUserAsync(ClaimsPrincipal user,
+            IQueryable<Notification> notifications)
+        {
+            _userPermissionsFilter ??= await GetUserPermissionsFilterAsync(user);
+            return _userPermissionsFilter.Type == UserType.NationalTeam
+                ? Enumerable.Empty<Notification>().AsQueryable()
+                : notifications.Where(n =>
+                    _userPermissionsFilter.IncludedTBServiceCodes.Contains(n.HospitalDetails.SecondaryTBServiceCode)
+                    || _userPermissionsFilter.IncludedPHECCodes.Contains(n.HospitalDetails.SecondaryTBService.PHECCode));

```

Comment:
Do we also need to check that the user doesn't have direct permissions on the notification?

Reply:
Ah, I think in my mind this section was only for notifications shared with you. Should the section include notifications in your TB service that are shared with others?

Reply:
I think we probably do want to make this change - if a user has 2 services, and the notification is owned by one and shared with the other, then when the user looks at the notification they will have full edit rights. I think it would be strange for the notification to appear in the list of shared notifications in this case.